test/unit/spec/userMenuSpec.js

```diff
@@ -122,20 +123,18 @@ define([
                         // expect to see the modal appear now
                         expect(document.querySelector(modalQuerySelector)).not.toBeNull();
 
-                        // click the appropriate button for this test
```

Comment:
This was actually a race condition that started failing after fixing other things.
The check for the style element on line 131 was failing, as it was already removed! That's the final thing that *should* happen, so this test now skips that intermediate part.